semantics.json

```diff
@@ -356,6 +357,13 @@
           "AudioRecorder"
         ]
       },
+      {
```

Comment:
When adding fields to semantics.json, remember to add these fields in the right places in every language file as well. Only `label` and `description` properties are needed in the language files.